src/Nest/Analysis/Analyzers/AnalyzerBase.cs

```diff
@@ -5,21 +5,18 @@ namespace Nest
 	[ContractJsonConverter(typeof(AnalyzerJsonConverter))]
 	public interface IAnalyzer
 	{
-		[JsonProperty(PropertyName = "version")]
+		[JsonProperty("version")]
 		string Version { get; set; }
 
-		[JsonProperty(PropertyName = "type")]
+		[JsonProperty("type")]
 		string Type { get; }
 	}
 
 	public abstract class AnalyzerBase : IAnalyzer
 	{
 		internal AnalyzerBase() { }
 
-		protected AnalyzerBase(string type)
-		{
-			Type = type;
-		}
+		protected AnalyzerBase(string type) => Type = type;
```

Comment:
I better upgrade Rider to 2017.2 and visual studio latest preview :) roll on c# 7.1